docs/rocketchat/get-help-in-rocketchat.md

```diff
@@ -1,95 +1,100 @@
+# Rocket.Chat help: developing stronger questions
+Last updated: **April 16, 2023**
+
+## On this page
```

Comment:
we don't need to manually do this part - this will automatically happen in the sidebar :)